src/main/java/io/cdap/plugin/http/sink/batch/HTTPRecordWriter.java

```diff
@@ -74,28 +78,36 @@ public class HTTPRecordWriter extends RecordWriter<StructuredRecord, StructuredR
   public void write(StructuredRecord input, StructuredRecord unused) throws IOException {
     String message = null;
     if (config.getMethod().equals("POST") || config.getMethod().equals("PUT")) {
-      if (config.getMessageFormat().equals("JSON")) {
-        message = StructuredRecordStringConverter.toJsonString(input);
-        contentType = "application/json";
-      } else if (config.getMessageFormat().equals("Form")) {
-        message = createFormMessage(input);
-        contentType = " application/x-www-form-urlencoded";
-      } else if (config.getMessageFormat().equals("Custom")) {
-        message = createCustomMessage(config.getBody(), input);
-        contentType = " text/plain";
-      }
-      messages.append(message).append(config.getDelimiterForMessages());
+        switch (config.getMessageFormat()) {
```

Comment:
fix indentation to be consistent (2 spaces)

Reply:
Updated